docs/user_guide/customizing.rst

```diff
@@ -7,49 +7,60 @@ Customizing the theme
 In addition to the configuration options detailed at :ref:`configuration`, it
 is also possible to customize the HTML layout and CSS style of the theme.
 
+.. _custom-css:
 
-Customizing the CSS
-===================
+Custom CSS Stylesheets
+======================
 
-The theme's CSS can be tweaked in 2 ways. The most straight forward way is to
-change the theme variables. If you are looking for more customisation, you can
-write additional css in ``custom.css``.
+You may customize the theme's CSS by creating a custom stylesheet that Sphinx uses to build your site.
+Any rules in this style-sheet will over-ride the default theme rules.
 
-Theme variables
----------------
+To add a custom stylesheet, follow these steps:
 
-This theme is based on top of the basic
-`Bootstrap CSS variables <https://getbootstrap.com/docs/4.0/getting-started/theming/#css-variables>`__
-extended with some theme specific variables. An overview of all variables and
-every default is defined in ``/pydata_sphinx_theme/static/css/theme.css``.
+1. **Create a CSS stylesheet** in ``_static/css/custom.css``, and add the CSS rules you wish.
+2. **Attach the stylesheet to your Sphinx build**. Add the following to ``conf.py``
 
-In order to change a variable, add a ``custom.css`` (see below) which updates
-the value of certain variables in a ``:root`` section:
+   .. code-block:: rst
 
-.. code-block:: none
+       html_static_path = ['_static']
 
-    :root {
-        --font-size-base: 17px;
-    }
+       html_css_files = [
+           'css/custom.css',
+       ]
 
-Important, the theme is defined with CSS variables, not SASS variables!
+When you build your documentation, this stylesheet should now be activated.
 
-Custom stylesheet
------------------
+CSS Theme variables
+===================
 
-If the theme variables are not sufficient to shape theme to you liking, you can
-take full control over the look and feel via a custom stylesheet. If you have a
-stylesheet in ``_static/css/custom.css``, adding the following to ``conf.py``
-will ensure this stylesheet is loaded last on top of the theme variables and the
-base styleheet:
+This theme defines several `CSS variables <css-variable-help_>`_ that can be
+used to quickly control behavior across your documentation.
+
+These are based on top of the basic `Bootstrap CSS variables <https://getbootstrap.com/docs/4.0/getting-started/theming/#css-variables>`_
+extended with some theme specific variables. An overview of all variables and
+every default is defined in `the pydata default CSS variables file <pydata-css-variables_>`_.
```

Comment:
Thanks a lot for updating this text. A link to the actual file with the variables is quite crucial to know which variables are available to override ..